docs/core/concepts/schedules.md

```diff
@@ -55,6 +55,12 @@ schedule = Schedule(clocks=[IntervalClock(timedelta(hours=24))])
 schedule.next(5)
 ```
 
+::: tip Time Zones
+Want to pin your schedule to a time zone? Specify a `start_date` corresponding to that time zone for your clock e.g.:
+
+`schedules.clocks.IntervalClock(start_date=pendulum.datetime(2019, 1, 1, tz='America/New York', interval=timedelta(days=1))`
```

Comment:
could we apply `black` to this just to add some spacing?

Reply:
Sure thing-- just updated, @cicdw.

Reply:
Awesome thanks!